From 715ee94b47f0f6d3af069c99293d33c440c26d66 Mon Sep 17 00:00:00 2001
From: Laacis <109668309+Laacis@users.noreply.github.com>
Date: Wed, 10 Jan 2024 08:14:45 +0800
Subject: [PATCH] Changed 2 lines in firstFile.txt in git_1

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..adb8b7d 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-1
-2
+a
+b
 3
 4
 5
-- 
2.42.0.windows.2

